README.rst

```diff
@@ -429,11 +429,27 @@ How to Cite
 
 If you utilize libCEED please cite::
 
-   @misc{libceed-dev-site,
-     title =  {lib{CEED} development site},
-     url =    {https://github.com/ceed/libceed},
-     howpublished = {\url{https://github.com/ceed/libceed}},
-     year = 2020
+   @misc{libceed-user-manual,
+     author       = {Abdelfattah, Ahmad and
+                     Barra, Valeria and
+                     Beams, Natalie and
+                     Brown, Jed and
+                     Camier, Jean-Sylvain and
+                     Dobrev, Veselin and
+                     Jean-Sylvain, Yohann and
```

Comment:
Is this name correct? Isn't it Yohann Dodouit?

Reply:
Thanks for the good catch! 

Weird this happened though. I downloaded the `bib` file as it is automatically generated on Zenodo.

@jedbrown was this list automatically generated from the [AUTHORS](https://github.com/CEED/libCEED/blob/main/AUTHORS) file? Or was this using the script for uploading to Zenodo?